2024/12/18 20:36:47 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.23.0.3:8080/", host: "localhost"
2024/12/18 20:36:47 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.23.0.3:8080/favicon.ico", host: "localhost", referrer: "http://localhost/"
